BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -43,6 +58,7 @@ class ProfileViewController: UIViewController {
         }
     }
     
+    // MARK:- Logout funtion
```

Comment:
Use showLogin function declared on UIViewController+extensions to launch login in one single line instead of create a scene delegate instance each time